app/Renderers/Layouts/Map/Marker.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Renderers\Layouts\Map;
+
+use App\Renderers\Layouts\Columns\Standard;
+use App\Renderers\Layouts\Layout;
+
+class Marker extends Layout
+{
+    /**
+     * Available columns
+     * @return array[]
+     */
+    public function columns(): array
+    {
+        $columns = [
+            'name' => [
+                'key' => 'name',
+                'label' => 'crud.fields.name',
+                'render' => function ($model) {
+                    return $model->tooltipedLink($model->markerTitle());
```

Comment:
tooltipedLink is used for links with tooltips and an ajax request on hover, so this name isn't consistent with the rest of the app. Also, using markerTitle() makes it that the order by name makes no sense 
![image](https://user-images.githubusercontent.com/3263440/197285732-1fe5a188-4853-4096-8762-d9938a2bb242.png)